### PR TITLE
Textarea component

### LINKED ISSUE #1357: Make (internal) Textarea component
Textarea is currently set as an `as` attribute inside `Input`. We need to move this logic into its own component to avoid lint errors and/or unnecessary calculations.

### PR DESCRIPTION
**Pull request by pomfrida**
Resolves #1357

### CODE REVIEW

File: libraries/core-react/src/components/Textarea/Textarea.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      disabled = false,
+      type = 'text',
+      rowsMax = 2,
+      ...other
+    },
+    ref,
+  ) {
+    const inputVariant = inputVariants[variant]
+    const spacings = tokens.comfortable.spacings
+    const [inputEl, setInputEl] = useState<HTMLTextAreaElement>(null)
```
Comment (mimarz) [inline libraries/core-react/src/components/Textarea/Textarea.tsx:113]: Suggest renaming this to appropriate name since its a `textarea` now 🤪

Comment (pomfrida) [reply libraries/core-react/src/components/Textarea/Textarea.tsx]: 👍  Done

**Review by mimarz — CHANGES_REQUESTED** [review]
We do have some cleaning up to do, but I suggest we can do some renaming so its easier to understand whats going on until then?

`InputWrapper.ts` -> `Field.ts` ?
`StyledInput` -> `StrippedInput`
`StyledTextarea` -> `StrippedTextArea`

File: libraries/core-react/src/components/Textarea/Textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+export const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
+  function Textarea(
+    {
+      variant = 'default',
+      disabled = false,
+      type = 'text',
+      rowsMax = 2,
+      ...other
+    },
+    ref,
```
Comment (mimarz) [inline libraries/core-react/src/components/Textarea/Textarea.tsx:109]: ref is not used, we need to combine this using the `useCombinedRef` hook

Comment (pomfrida) [reply libraries/core-react/src/components/Textarea/Textarea.tsx:109]: 👍  Done

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**